CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -1485,6 +1589,7 @@
       </trans-unit>
       <trans-unit id="What was that thing I looked up yesterday?" xml:space="preserve">
         <source>What was that thing I looked up yesterday?</source>
+        <target>Co to było, czego szukałam/em wczoraj?</target>
```

Comment:
"Czego wczoraj szukałam/em?"

Reply:
As this is a bit like someone talking to themself in English, I opted for "Czego to ja wczoraj szukałam/em?"

Yay or nay? ;)

Reply:
I like my idea better, but obviously it's up to you. I'm trying to make the strings short and sweet (definitely trying to not make them much longer than originals) because of size constraints and trying to be communicative without forcing the user to read too much.